### PR TITLE
docs(flutterinfinitelist): add stream_transform import for `throttle` api

### PR DESCRIPTION
**Pull request by MayurSMahajan**
## Status

**READY**

## Breaking Changes

NO

## Description

Solves  #3556 

While following along the documentation, in the section of PostBloc, the docs introduce the  `events.throttle(duration)` method. This method requires the **stream_transform/stream_transform.dart** package. But the package import statement is not in the code snippet, also there is no mention that this package is required to run the `events.throttle(duration)` method. 

The next code snippet does contain the import statement, but naive developers like me who are following along get stuck. We see the error and don't know what to do. Therefore I think it is necessary that there is some mention to the package import. Thus my Pull Request adds a note instruction and updates the code snippet to include the necessary changes.

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: docs/flutterinfinitelisttutorial.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ If we can retrieve the posts, we return `PostSuccess()` which takes the entire l
 One optimization we can make is to `debounce` the `Events` in order to prevent spamming our API unnecessarily. We can do this by overriding the `transform` method in our `PostBloc`.
 
 ?> **Note:** Passing a `transformer` to `on<PostFetched>` allows us to customize how events are processed.
+?> **Note:** Make sure to import the stream_transform package in order to use the `events.throttle(duration)` method.
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:150]:
```suggestion
?> **Note:** Make sure to import [`package:stream_transform`](https://pub.dev/packages/stream_transformer) the `throttle` api.
```

Comment (MayurSMahajan) [reply docs/flutterinfinitelisttutorial.md]: Done!

File: docs/flutterinfinitelisttutorial.md [hunk 2]
```diff
@@ -147,6 +147,7 @@ If we can retrieve the posts, we return `PostSuccess()` which takes the entire l
 One optimization we can make is to `debounce` the `Events` in order to prevent spamming our API unnecessarily. We can do this by overriding the `transform` method in our `PostBloc`.
 
 ?> **Note:** Passing a `transformer` to `on<PostFetched>` allows us to customize how events are processed.
+?> **Note:** Make sure to import [`package:stream_transform`](https://pub.dev/packages/stream_transformer) the `throttle` api.
```
Comment (felangel) [inline docs/flutterinfinitelisttutorial.md:150]:
```suggestion
?> **Note:** Make sure to import [`package:stream_transform`](https://pub.dev/packages/stream_transformer) to use the `throttle` api.
```

**Review by felangel — APPROVED** [review]

**Comment by felangel** [comment]
> Hey Felix! I just realized that the URL to the stream transform package is https://pfasdklfjasdkfklfasdf and not fksasklfsklfnasdf
> 
> image
> 
> honest mistake 😅
> 
> image
> 
> should I create a new pr or what?

No worries I just updated the URL. Thanks!

**Pull request merged**